### DISCUSSION TITLE
Zed AI: Account Too Young?

### DISCUSSION THREAD

**Discussion by geoffreywiseman** [zed-industries/zed#16561] [Feature Requests]
Trying out "Zed AI", I get:
> Error interacting with language model
> RPC request GetLImToken failed: account too young
>
> Dismiss

I get that this is trying to tell me something, but not really sure what, or how to fix it, if it can be fixed?  Google search didn't get me anything, nor did a quick search of the discussions, so I figured I should ask.

**Comment by SomeoneToIgnore** [comment]
Zed uses a heuristic to protect from spam accounts, setting a "at least 30 days old" requirement on the accounts.
So, unless there's some bug with a specific account metadata parsing, that should solve itself in a month.

**Reply by ashwanthkumar** [reply]
To use the feature that was released yesterday we need to wait 30 days or does this account mean something else?

**Reply by SomeoneToIgnore** [reply]
I mean that whatever GitHub account is used to access Zed features, has to be 30 days old or more.

**Reply by rlrs** [reply]
Could this be Github account age? If it's some hidden Zed account you're scaring away all new users interested in the AI features. Edit: I see that it's supposed to be, but I get the same message and my Github account is way older.

**Reply by Alaz-Oz** [reply]
I too got that error, and my github account is very old that I am using with the zed. I installed the zed 2-3 weeks before and accepted the terms and conditions for the Zed AI, today.

**Reply by klarsan** [reply]
In my situation I just had to log into zed.dev with github account and then log into application itself. Then additional LLM appeared - "Claude 3.5 Sonnet" with the addition of "zed". I had 1 "Claude 3.5 Sonnet" before first one had addition of "Anthropic".

**Comment by SomeoneToIgnore** [comment]
And I've just got a confirmation from someone who worked with this, that there's a bug that's being investigated.

Current workaround for old enough accounts with this error popping-up is to go to zed.dev, sign out, sign back in again and try accessing the feature once more.

**Reply by rlrs** [reply]
This solved it for me.

**Reply by mdpstory** [reply]
its not working for me

**Comment by JosephTLyons** [comment]
The workaround proposed here should work, but we are also currently fixing this problem for all previous Zed users, such that only accounts that are actually too young will hit this limitation.

**Reply by ghost** [reply]
how young is too young please ?
